src/PointwiseFunctions/AnalyticData/Xcts/Binary.cpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/Xcts/Binary.hpp"
+
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "PointwiseFunctions/AnalyticData/Xcts/CommonVariables.tpp"
+#include "PointwiseFunctions/Elasticity/Strain.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Xcts/LongitudinalOperator.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// \cond
+namespace Xcts::AnalyticData::detail {
+
+template <typename DataType>
+void BinaryVariables<DataType>::add_deriv_of_window_function(
+    const gsl::not_null<tnsr::ijj<DataType, Dim>*> deriv_conformal_metric)
+    const noexcept {
+  if (not falloff_widths.has_value()) {
+    return;
+  }
+  const auto& flat_metric =
+      get<Tags::ConformalMetric<DataType, 3, Frame::Inertial>>(flat_vars);
+  for (size_t i = 0; i < 3; ++i) {
+    for (size_t j = 0; j < 3; ++j) {
+      for (size_t k = 0; k <= j; ++k) {
+        for (size_t alpha = 0; alpha < 2; ++alpha) {
+          deriv_conformal_metric->get(i, j, k) -=
+              2. * gsl::at(x_isolated, alpha).get(i) /
+              square(gsl::at(*falloff_widths, alpha)) *
+              gsl::at(windows, alpha) *
+              (get<Tags::ConformalMetric<DataType, 3, Frame::Inertial>>(
+                   gsl::at(isolated_vars, alpha))
+                   .get(j, k) -
+               flat_metric.get(j, k));
+        }
+      }
+    }
+  }
+}
+
+template <typename DataType>
+void BinaryVariables<DataType>::operator()(
+    const gsl::not_null<tnsr::I<DataType, Dim>*> shift_background,
+    const gsl::not_null<Cache*> /*cache*/,
+    Tags::ShiftBackground<DataType, Dim, Frame::Inertial> /*meta*/)
+    const noexcept {
+  get<0>(*shift_background) = -angular_velocity * get<1>(x);
+  get<1>(*shift_background) = angular_velocity * get<0>(x);
+  get<2>(*shift_background) = 0.;
+}
+
+template <typename DataType>
+void BinaryVariables<DataType>::operator()(
+    const gsl::not_null<tnsr::iJ<DataType, Dim>*> deriv_shift_background,
+    const gsl::not_null<Cache*> /*cache*/,
+    ::Tags::deriv<Tags::ShiftBackground<DataType, Dim, Frame::Inertial>,
+                  tmpl::size_t<Dim>, Frame::Inertial> /*meta*/) const noexcept {
+  std::fill(deriv_shift_background->begin(), deriv_shift_background->end(), 0.);
+  get<1, 0>(*deriv_shift_background) = -angular_velocity;
+  get<0, 1>(*deriv_shift_background) = angular_velocity;
+}
+
+template <typename DataType>
+void BinaryVariables<DataType>::operator()(
```

Comment:
I considered that, but it requires all solutions implement `deriv(ShiftBackground)` _and_ `dt(ConformalMetric)` instead. In most cases it's easier to just set the longitudinal-shift-minus-dt(gamma) to zero. In fact, I can't think of another scenario right now where it's non-zero. That said, it may turn out to be useful to move this to `CommonVariables` at some point in the future, but not right now I think.